### PR TITLE
Enabled IntelliSense Documentation

### PR DESCRIPTION
**Pull request by indy-singh**
Issue: https://github.com/Microsoft/Microsoft.IO.RecyclableMemoryStream/issues/53

### CODE REVIEW

**Comment by doubleyewdee** [comment]
Shouldn't this be bin\$(Configuration)\...xml?

**Comment by Singh400** [comment]
@doubleyewdee I don't know to be honest. I just ticked the option in the settings of the CSProj file. I can amend the PR.

Cheers,
Indy

**Comment by Singh400** [comment]
Hi @doubleyewdee,

Is there anything I can do to get this accepted? We have started to develop features around RMS around would find IntelliSense Documentation very helpful.

Thanks,
Indy

**Comment by indy-singh** [comment]
Ping @doubleyewdee

File: src/Microsoft.IO.RecyclableMemoryStream.csproj [hunk 1]
```diff
@@ -10,4 +10,13 @@
     <DelaySign>true</DelaySign>
     <SignAssembly>true</SignAssembly>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard1.4|AnyCPU'">
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Microsoft.IO.RecyclableMemoryStream.xml</DocumentationFile>
```
Comment (benmwatson) [inline src/Microsoft.IO.RecyclableMemoryStream.csproj]: Why not just add this property without conditions--the value is always the same

**Comment by benmwatson** [comment]
@indy-singh I left a comment--I don't think that you need the conditionals with duplicate properties. A single copy of that property should suffice.

File: src/Microsoft.IO.RecyclableMemoryStream.csproj (hunk 1 above)
Comment (indy-singh) [reply src/Microsoft.IO.RecyclableMemoryStream.csproj]: Agreed. I'll try and get this done in the next couple of days.

**Comment by benmwatson** [comment]
I am accomplishing this in #82

**Pull request closed without merging**